tests/phpunit/Mutator/Number/IncrementIntegerTest.php

```diff
@@ -258,5 +259,39 @@ public function mutationsProvider(): iterable
 random_int(10000001, {$maxInt});
 PHP
         ];
+
+        $minIntPlus1 = PHP_INT_MIN + 1;
+        $minIntPlus2 = $minIntPlus1 + 1;
+
+        yield 'It increments min int plus one, -PHP_INT_MAX' => [
+            <<<"PHP"
+            <?php
+                        
```

Comment:
Going to disable trailing whitespace check since it jumps on this extra whitespace inside strings: we can't remove it.
```
Trailing whitespaces detected:
 - in tests/phpunit/Mutator/Number/IncrementIntegerTest.php at line 269
   > 
 - in tests/phpunit/Mutator/Number/IncrementIntegerTest.php at line 277
   > 
 - in tests/phpunit/Mutator/Number/IncrementIntegerTest.php at line 290
make[1]: *** [Makefile:65: check_trailing_whitespaces] Error 1
   > 
 - in tests/phpunit/Mutator/Number/DecrementIntegerTest.php at line 528
   > 
 - in tests/phpunit/Mutator/Number/DecrementIntegerTest.php at line 535
   > 
```